### ISSUE TITLE
[Feature] introduce Log-Level warning for flux-extension to get "skipped-entry" info as warning instead of info/debug

### ISSUE THREAD

**Issue by pandvag** [Azure/AKS#4575]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Actually flux extension supports only three log-level: info/error/debug
Because info/debug writes a lot of logs and only prints certain error logs, you have no other way to minimize logging and still be able to see skipped messages (kustomize.toolkit.fluxcd.io/reconcile: disabled).

**Describe alternatives you've considered**
An ideal solution would be to introduce log-level **_warning_** that also prints skipped messages. This way customers can minimize the log volume, but at the same time still see all relevant messages that do not conform to the norm like skipped entries.

**Additional context**

**Comment by dipti-pai** [comment]
Changing the assignment, this was missed as it was incorrectly assigned. 

@pandvag If you are still looking for this feature, could you share an example of a log statement ? Thanks.

**Comment by pandvag** [comment]
Hi @dipti-pai 
thanks for your reply. 

Actually within kustomize-controller all objects who have "kustomize.toolkit.fluxcd.io/reconcile: disabled" defined are marked as "skipped", for instance log is generated as 

> "Ingress/blabla":"skipped","Ingress/blabla2":"skipped",

**Comment by dipti-pai** [comment]
Thanks @pandvag. The log type for this statement would need to be changed in kustomize-controller and cannot be controlled from  the flux-extension.

**Comment by pandvag** [comment]
yes you are right @dipti-pai, it is related to kustomize-controller within the flux-extension. If you are using "error" logging in kustomize-controller you won't see any "skipped" messages, that's what this issue is about

**Comment by dipti-pai** [comment]
The debug/info/error logging in Flux controllers are consistent with upstream Flux. Any enhancement would need to be made upstream.
